ui/components/src/GaugeChart/GaugeChart.md

```diff
@@ -0,0 +1,17 @@
+# GaugeChart
```

Comment:
I could see that getting really long as we add more components, maybe having a generic example in ui/components/README.md and then links to each individual component's doc could work? 

Having a docs file alongside each component will also work well with Storybook in the future since it automatically [pulls in that info](https://storybook.js.org/docs/react/api/mdx#basic-example) (will just change from .md to .mdx).

Reply:
yeah I think having at least the list of the component in the README at the root of the package would be cool.